src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/index.jelly

```diff
@@ -243,6 +248,11 @@ THE SOFTWARE.
                 You can also remove all previous signature approvals:
                 <button onclick="if (confirm('Really delete all approvals? Any existing scripts will need to be rerun and signatures reapproved.')) {clearApprovedSignatures()}">Clear Approvals</button>
             </p>
+            <j:if test="${!empty(dangerousApprovedSignatures)}">
+                Or you can just remove the dangerous ones:
+                <button onclick="if (confirm('Really delete all dangerous approvals? Any scripts that was using such signatures will need to be rerun and signatures reapproved.')) {clearDangerousApprovedSignatures()}">Clear only dangerous Approvals</button>
```

Comment:
Certainly the confirmation message should be reworded, and possibly no confirmation is even needed for this case.